docs/01-getting-started/01-installation.mdx

```diff
@@ -123,12 +129,64 @@ export default function Page() {
 
 > **Good to know**: If you forget to create `layout.tsx`, Next.js will automatically create this file for you when running the development server with `next dev`.
 
-### Create the `public` folder
+Learn more about [using the App Router](https://nextjs.org/docs/app/building-your-application/routing/defining-routes).
+
+#### The `pages` folder (optional)
+
+If you prefer to use the older `pages` router instead of `app`, you can create a `pages` folder at the root of your project.
+
+Then add an `index.tsx` file inside your pages folder, this will be your home page:
+
+```tsx filename="pages/index.tsx" switcher
+export default function Page() {
+  return <h1>Hello, Next.js!</h1>
+}
+```
+
+Next, add a custom `_app` file inside your pages folder. Learn more about the [custom App file](/docs/pages/building-your-application/routing/custom-app)).
+
+```tsx filename="pages/_app.tsx" switcher
+import type { AppProps } from 'next/app'
+
+export default function App({ Component, pageProps }: AppProps) {
+  return <Component {...pageProps} />
+}
+```
+
+```jsx filename="pages/_app.js" switcher
+export default function App({ Component, pageProps }) {
+  return <Component {...pageProps} />
+}
+```
+
+Finally, add a custom `_document` file inside your pages folder. Learn more about the [custom Document file](/docs/pages/building-your-application/routing/custom-document).
+
+```tsx filename="pages/_document.tsx"
+import { Html, Head, Main, NextScript } from 'next/document'
+
+export default function Document() {
+  return (
+    <Html>
+      <Head />
+      <body>
+        <Main />
+        <NextScript />
+      </body>
+    </Html>
+  )
+}
+```
+
+Learn more about [using the Pages Router](/docs/pages/building-your-application/routing/pages-and-layouts).
+
+> **Good to know**: Although you can use both routers in the same project, routes in `app` will be prioritized over `pages`. We recommend using only one router in your new project to avoid confusion.
```

Comment:
```suggestion
> **Good to know**: Although you can use both routers in the same project, routes in `app` will be prioritized over `pages`. Learn more about [incrementally adopting the App Router](/docs/app/building-your-application/upgrading/app-router-migration) if you have an existing project.
```

Not sure on this... maybe not necessary since it's the installation page?

Reply:
Yes, this is for new apps.